tests/invalid_models_tests/test_relative_fields.py

```diff
@@ -132,6 +133,45 @@ class Model(models.Model):
         ]
         self.assertEqual(errors, expected)
 
+    def test_foreign_key_to_isolated_apps_model(self):
+        test_apps = Apps(['invalid_models_tests'])
+
+        class OtherModel(models.Model):
+            class Meta:
+                apps = test_apps
+
+        class Model(models.Model):
+            foreign_key = models.ForeignKey('OtherModel', models.CASCADE)
+
+            class Meta:
+                apps = test_apps
+
+        field = Model._meta.get_field('foreign_key')
+        self.assertEqual(field.check(from_model=Model), [])
+
+    def test_foreign_key_to_other_apps_model(self):
+        other_apps = Apps(['invalid_models_tests'])
+
+        class OtherModel(models.Model):
+            class Meta:
+                apps = other_apps
+
+        class Model(models.Model):
+            foreign_key = models.ForeignKey(OtherModel, models.CASCADE)
+
+        field = Model._meta.get_field('foreign_key')
+        errors = field.check()
+        expected = [
+            Error(
+                ("Field defines a relation with model 'OtherModel', "
```

Comment:
I would omit the parenthesis in these messages (I know it's done elsewhere, but "I am at war" with that style unless you like it).